Digitall.Warehouse.Domain/Entities/Products/Product.cs

```diff
@@ -0,0 +1,23 @@
+﻿using Digitall.Warehouse.Domain.Abstraction;
+
+namespace Digitall.Warehouse.Domain.Entities.Products
+{
+    public class Product : IIdentifiable<Guid>
+    {
+        public Guid Id { get; }
+
+        public string Title { get; set; } = null!;
+
+        public string? Description { get; set; }
+
```

Comment:
CodeMaid is good extension for VS that can check the files for empty rows and beautifying the code.

Reply:
Thanks will take a look :).
Does replace the VS built-in short keys for refactoring and formatting?